keyboards/singa/singa.c

```diff
@@ -34,6 +34,25 @@ void rgblight_set(void) {
 }
 #endif
 
+void matrix_init_kb(void) {
+#ifdef RGBLIGHT_ENABLE
+    if (rgblight_config.enable) {
```

Comment:
Can `rgblight_set()` just be called in place of lines 39-42?

Reply:
Probably fine, but `jj4x4`, `alice`, and the `ps2avrgb` template use a different format that is slightly different than this. Do you prefer the existing pattern in the template, or to call `rgblight_set()`?

Reply:
Actually, it may be better to move this entire block to `keyboard_post_init_kb` instead.